comp/swerve100/src/main/java/org/team100/frc2024/RobotContainer.java

```diff
@@ -196,10 +197,13 @@ public RobotContainer(TimedRobot robot) throws IOException {
         m_feeder = new FeederSubsystem(39);
 
         m_intake = IntakeFactory.get(m_sensors);
+        //TODO get real strips that are front and back
+        List<LEDStrip> strips = new ArrayList<>();
+        for (int i = 0; i< 10;i++) {
+            strips.add(new LEDStrip(16, i*16));
+        }
 
-        LEDStrip strip1 = new LEDStrip(160, 0);
-
-        m_indicator = new LEDIndicator(0, strip1);
+        m_indicator = new LEDIndicator(0, 2, strips.get(0),strips.get(1), strips.get(2), strips.get(3), strips.get(4), strips.get(5), strips.get(6), strips.get(7), strips.get(8), strips.get(9));
```

Comment:
i think you really mean to just pass the whole "strips" list as an array, which you can do with:

strips.toArray(new LedStrip[0])